.gitignore

```diff
@@ -4,6 +4,7 @@ venv
 **/__pycache__
 .pytest_cache
 bin/
+bpdvenv/
```

Comment:
```suggestion
```

I get this is your local environment but we shouldn't start a precedent of adding custom envfiles

Reply:
Yes, thats right